app/src/main/java/com/onboarding/wheatherapp/services/response/MainResponse.kt

```diff
@@ -0,0 +1,7 @@
+package com.onboarding.wheatherapp.services.response
+
+data class MainResponse(
+    val temp: String,
+    val temp_min: String,
```

Comment:
the name of this var don't respect android's naming convention please use
@SerializedName tag
for example
@SerializedName("temp_min") val tempMin: Double

Reply:
Solved!